providers/mattermost/src/lib/mattermost.provider.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { MattermostProvider } from './mattermost.provider';
+import axios from 'axios';
```

Comment:
Please put the dependency imports first in the file.

Reply:
Updated